Adyen/Model/PaymentComponentData.swift

```diff
@@ -13,6 +13,8 @@ import Foundation
  [API Reference](https://docs.adyen.com/api-explorer/#/CheckoutService/latest/post/payments__example_payments-klarna)
  */
 public struct PaymentComponentData {
+
+    internal let internalAmount: Amount?
```

Comment:
maybe `totalAmount` makes more sense?

Reply:
Since we introduced `amountToPay` - we can just use `amount` 🤔